app/workers/central_mail/submit_form4142_job.rb

```diff
@@ -4,9 +4,23 @@
 require 'common/exceptions'
 require 'evss/disability_compensation_form/metrics'
 require 'evss/disability_compensation_form/form4142_processor'
+require 'logging/third_party_transaction'
 
 module CentralMail
   class SubmitForm4142Job < EVSS::DisabilityCompensationForm::Job
+    extend Logging::ThirdPartyTransaction::MethodWrapper
+
+    # this is required to make instance variables available to logs via
+    # thw wrap_with_logging method
```

Comment:
nit pick, I think "thw" is a typo and not some new fangled abbreviation all the cool coders are using these days?

Reply:
fixed : )